gensim/models/keyedvectors.py

```diff
@@ -1052,15 +1193,16 @@ def _log_evaluate_word_analogies(section):
             logger.info("%s: %.1f%% (%i/%i)", section['section'], 100.0 * score, correct, correct + incorrect)
             return score
 
-    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True, dummy4unknown=False):
+    def evaluate_word_analogies(self, analogies, restrict_vocab=300000, case_insensitive=True,
+    dummy4unknown=False, topk=5, method='3CosAdd'):
```

Comment:
Start the params on a new line, with an extra indent level.

Reply:
Fixed